spec/system/page_objects/modals/sidebar_section_form.rb

```diff
@@ -12,6 +12,10 @@ def fill_link(name, url)
         fill_in "link-url", with: url, match: :first
       end
 
+      def mark_as_public
+        find(".modal .checkbox-label").click
```

Comment:
Should probably do a more specific selector than this, include an ID or something?